tests/acceptance_crontab_spec.lua

```diff
@@ -0,0 +1,216 @@
+            vim.cmd("CronExplainedDisable")
+            
+            -- Reset buffer to original
+            vim.api.nvim_set_current_buf(orig_buf)
+            pcall(vim.api.nvim_buf_delete, unquoted_buf, { force = true })
+        end)
+        
+        it("Recognizes unquoted crontab expressions", function()
+            -- Setup plugin with custom crontab extractor
+            cronex.setup({
+                file_patterns = { "*.crontab" },
+                extractor = {
+                    -- This will be implemented in our custom extractor
+                    cron_from_line = require("cronex.cron_from_line").cron_from_line_crontab,
+                    extract = require("cronex.extract").extract
+                },
+                explainer = {
+                    cmd = { "echo", "test-explanation" }
+                }
+            })
+            
+            -- Set current buffer and enable plugin
+            vim.api.nvim_set_current_buf(unquoted_buf)
+            vim.cmd("CronExplainedEnable")
+            
+            -- Wait for async operations to complete
+            wait()
+            
+            -- Get diagnostics
+            local diags = vim.diagnostic.get(unquoted_buf)
+            assert.is_not_nil(diags, "Diagnostics should exist")
+            assert.is_true(#diags > 0, "Should have diagnostics for cron lines")
+            
+            -- Should have diagnostics for all cron lines (not comments)
+            assert.are.equal(7, #diags, "Should have 7 diagnostics for all cron lines")
+            
+            -- Verify all lines have explanations
+            local line_has_diag = {}
+            for i = 0, 7 do
+                line_has_diag[i] = false
+            end
+            
+            for _, diag in ipairs(diags) do
+                line_has_diag[diag.lnum] = true
+            end
+            
+            -- Standard 5-part cron expressions
+            assert.is_true(line_has_diag[0], "Should recognize '* * * * *' pattern")
+            assert.is_true(line_has_diag[1], "Should recognize '30 5 * * 1-5' pattern")
+            assert.is_false(line_has_diag[2], "Should ignore comment lines")
+            assert.is_true(line_has_diag[3], "Should recognize '@daily' special syntax")
+            assert.is_true(line_has_diag[4], "Should recognize '0 */4 * * *' pattern")
+            assert.is_true(line_has_diag[5], "Should only match first 5 parts")
+            assert.is_true(line_has_diag[7], "Should recognize 6-part cron")
+            
+            -- Verify that 6-part cron is recognized correctly
+            for _, diag in ipairs(diags) do
+                if diag.lnum == 7 then
+                    local expected_cron = "1 2 3 4 5" -- 5 parts, ignoring seconds
+                    assert.truthy(string.match(diag.message, "test%-explanation"))
+                end
+            end
+        end)
+        
+        it("Correctly parses 6-part cron expressions", function()
```

Comment:
this last `it` is actually more of a unit test (not acceptance), since we are checking the functionality of `cron_from_line_crontab`.
It would be wonderful if we could have it in a separate file `cron_from_file_crontab_spec.lua` (analogous to [this](https://github.com/fabridamicelli/cronex.nvim/blob/main/tests/cron_from_line_spec.lua))

Also, it'd be great to have more extensive examples  (similar to  [this](https://github.com/fabridamicelli/cronex.nvim/blob/main/tests/cron_from_line_spec.lua) to make sure that the regex patterns are working fine on edge cases as well